app.base.json

```diff
@@ -32,6 +32,8 @@
     "footer1": "Bel ons dan",
     "footer2": "op werkdagen van 9.00 tot 17.00 uur.",
     "headerTitle": "",
+    "helpText": "Bereikbaar van maandag tot en met vrijdag van 08:00 tot 18:00 uur.",
+    "helpTextHeader": "Hulp nodig: Bel het telefoonnummer [14020](tel:14020)",
```

Comment:
Adding this line will require each municipality to do the same. What do you think of this proposed change, @boris-arkenaar ?

Reply:
If I understand correctly, what every municipality should do is rename `footer1` and `footer2` to `helpTextHeader` and `helpText` respectivily.